Source/Core/DolphinQt/NetPlay/ClickBlurLabel.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2025 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <QGraphicsBlurEffect>
+#include <QLabel>
+#include <QStackedWidget>
+#include <QWidget>
+
+class ClickBlurLabel final : public QWidget
+{
+  Q_OBJECT
+public:
+  explicit ClickBlurLabel(QWidget* parent = nullptr);
+  explicit ClickBlurLabel(const QString& text, QWidget* parent = nullptr);
+
+  void setText(const QString& text);
+
+  QString text() const { return m_normal_label->text(); }
+
+protected:
+  void mousePressEvent(QMouseEvent* event) override;
+
+private:
+  // Generates text that "looks correct" but is actually gibberish.
+  QString GenerateBlurredText(const QString& text);
```

Comment:
```suggestion
  static QString GenerateBlurredText(const QString& text);
```
Eliminates a clang-tidy warning.